src/EditorFeatures/TestUtilities/Extensions/WorkspaceExtensions.cs

```diff
@@ -2,21 +2,30 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
+
 using System.Collections.Generic;
+using System.Linq;
 using Microsoft.CodeAnalysis.Options;
 
 namespace Microsoft.CodeAnalysis.Editor.UnitTests.Extensions
 {
     public static class WorkspaceExtensions
     {
-        public static void ApplyOptions(this Workspace workspace, IDictionary<OptionKey, object> options)
+        public static void ApplyOptions(this Workspace workspace, IDictionary<OptionKey, object>? options)
+            => workspace.ApplyOptions(options.Select(kvp => (kvp.Key, kvp.Value)));
+
+        internal static void ApplyOptions(this Workspace workspace, IDictionary<OptionKey2, object>? options)
+            => workspace.ApplyOptions(options.Select(kvp => ((OptionKey)kvp.Key, kvp.Value)));
```

Comment:
❕ `options` needs to be non-nullable, or the implementation needs to account for it in the call to `Select`.